packages/header-app/src/components/openhim.appbar.component.tsx

```diff
@@ -117,15 +126,27 @@ export default function OpenhimAppBar() {
   const [isAdmin, setIsAdmin] = useState(false)
   const [anchorElNav, setAnchorElNav] = useState<null | HTMLElement>(null)
   const [anchorElUser, setAnchorElUser] = useState<null | HTMLElement>(null)
```

Comment:
Is this constant different from `const [anchorElUsers, setAnchorElUsers]`

Reply:
Yes. Each dropdown menu item needs to have its own anchor element state.